packages/worker/src/utilities/appService.ts

```diff
@@ -20,11 +21,11 @@ async function makeAppRequest(url: string, method: string, body: any) {
   return fetch(checkSlashesInUrl(env.APPS_URL + url), request)
 }
 
-export async function syncUserInApps(userId: string) {
+export async function syncUserInApps(userId: string, previousUser?: User) {
   const response = await makeAppRequest(
     `/api/users/metadata/sync/${userId}`,
     "POST",
-    {}
+    { previousUser }
```

Comment:
With the new type definition from above we can add

```ts
const body: SyncUserRequest = {
   previousUser
}

  const response = await makeAppRequest(
    `/api/users/metadata/sync/${userId}`,
    "POST",
    body
```

So that this api request will always align with future changes to the type definition